streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregate.java

```diff
@@ -48,15 +51,25 @@
     private final Windows<W> windows;
     private final Initializer<VAgg> initializer;
     private final Aggregator<? super KIn, ? super VIn, VAgg> aggregator;
+    private final boolean emitFinal;
 
     private boolean sendOldValues = false;
 
     public KStreamWindowAggregate(final Windows<W> windows,
                                   final String storeName,
                                   final Initializer<VAgg> initializer,
                                   final Aggregator<? super KIn, ? super VIn, VAgg> aggregator) {
+        this(windows, storeName, false, initializer, aggregator);
+    }
+
+    public KStreamWindowAggregate(final Windows<W> windows,
```

Comment:
The PR title says "sliding window" but this it the "time window" class (ie, hopping/tumbling).